.github/workflows/test.yaml

```diff
@@ -20,9 +20,9 @@ jobs:
         with:
           go-version-file: "go.mod"
 
-      - uses: aquaproj/aqua-installer@v2.1.2
+      - uses: aquaproj/aqua-installer@v3.0.0
         with:
-          aqua_version: v2.10.1
+          aqua_version: v2.27.0
```

Comment:
I got a error and resolved by update aqua: https://github.com/hirosassa/tfcmt-gitlab/actions/runs/8629098634/job/23652549544